mvvmfx/src/main/java/de/saxsys/mvvmfx/utils/dialogs/DialogHelper.java

```diff
@@ -0,0 +1,63 @@
+package de.saxsys.mvvmfx.utils.dialogs;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.Window;
+
+import java.util.function.Supplier;
+
+/**
+ * Created by Sven on 21/06/16.
+ */
+public class DialogHelper {
+
+	// prevent instantiation from outside
+	private DialogHelper() {}
+
+	public static Stage initDialog(Window owner, Supplier<Parent> content, BooleanProperty visibleProperty, String...
```

Comment:
I think we should find a better name for "visibleProperty". In the original DialogHelper the name "openProperty" was used which isn't really a better name. 
May "showProperty"? It's what actually is happening (stage.show())